.travis.yml

```diff
@@ -4,10 +4,7 @@ php:
   - '5.6'
   - '7.0'
 install:
-  - composer install
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer update --no-interaction
```

Comment:
I could keep the lock file in version control but I can not use composer update instead of install.